source/fundamentals/transactions.txt

```diff
@@ -71,25 +72,42 @@ following table describes these methods:
      - | Starts a new transaction, configured with the given options, on
          this session. Returns an error if there is already
          a transaction in progress for the session. For more
-         information, see the :manual:`manual entry </reference/method/Session.startTransaction/>`.
+         information, see :manual:`startTransaction()
+         </reference/method/Session.startTransaction/>`  in the Server manual.
        |
        | **Parameter**: ``TransactionOptions``
        | **Return Type**: ``error``
 
    * - ``AbortTransaction()``
-     - | Aborts the active transaction for this session. Returns an
+     - | Ends the active transaction for this session. Returns an
          error if there is no active transaction for the session or the
-         transaction has been committed or aborted. For more
-         information, see the :manual:`manual entry </reference/method/Session.abortTransaction/>`.
+         transaction has been committed or ended. For more
+         information, see :manual:`abortTransaction()
+         </reference/method/Session.abortTransaction/>` in the Server manual.
        |
        | **Parameter**: ``Context``
        | **Return Type**: ``error``
 
    * - ``CommitTransaction()``
      - | Commits the active transaction for this session. Returns an
-         error if there is no active transaction for the session or the
-         transaction has been aborted. For more
-         information, see the :manual:`manual entry </reference/method/Session.commitTransaction/>`.
+         error if there is no active transaction for the session or if the
+         transaction was ended. For more
+         information, see :manual:`commitTransaction()
+         </reference/method/Session.commitTransaction/>` in the Server
+         manual.
+         
+         .. note:: Retrying a Transaction
+            
+            The ``CommitTransaction()`` method is an idempotent function, which
```

Comment:
CommitTransaction appears idempotent in experimentation, but I can't find server documentation supporting this. Can we assign someone on the server team to review this PR to ensure this is a true statement?

Reply:
Sure - I misunderstood the ticket. I can ask a server engineer to review and confirm.